sprocs/question_order.sql

```diff
@@ -70,7 +70,11 @@ WINDOW
                         WHEN a.shuffle_questions THEN iq.order_by
                         ELSE aq.number
                     END
-                WHEN a.type = 'Exam' THEN iq.order_by
+                WHEN a.type = 'Exam' THEN
+                    CASE
+                        WHEN a.shuffle_questions THEN iq.order_by
+                        ELSE aq.number
+                    END
```

Comment:
Let's collapse this into the previous case, e.g.

```sql
WHEN (a.type = 'Homework' OR a.type = 'Exam') THEN ...
```

In practice, we should only ever have this case, but it looks like there are some other (probably unsupported) assessment types from the deep past that I don't want to risk breaking: https://github.com/PrairieLearn/PrairieLearn/blame/15a01e6b15c520f5e0435ced1a1fb0d5bfb8aefd/database/enums/enum_assessment_type.pg

Reply:
Done in [#40e6ff5](https://github.com/PrairieLearn/PrairieLearn/pull/5892/commits/40e6ff57456d3d2e36903d9b848ca27c6be91c72).